environment/src/main/java/com/sequenceiq/environment/parameters/dao/domain/AzureParameters.java

```diff
@@ -36,6 +36,9 @@
     @SecretValue
     private Secret encryptionKeyUrl = Secret.EMPTY;
 
+    @Column(name = "disk_encryption_set_id")
+    private String diskEncryptionSetId;
+
```

Comment:
As mentioned earlier, we probably will have to add one more field & DB column here for the Key Vault RG name or Resource ID.

Reply:
Do we really need to store Key Vault RG name or Resource ID ? 
`Key Vault RG name` is required during creation flow in case of multi-RG. Not sure if we would need it going forward.
If we decide to store it, would it make sense to (fetch and) store it during EnvironmentCreationDto from EnvironmentRequest ?

Reply:
Yes, I agree that it is currently needed only once during the DES creation flow step, but storing it would still make sense to keep track of all user input given during environment creation.

Please note, however, that the Key Vault RG (or Resource ID) is always needed, irrespective of Single-RG vs Multi-RG. The Key Vault may be located in a totally different RG even when using Single-RG; that is what my setup looks like, and customers may decide to follow this approach as well in order to separate persistent resources (e.g. Key Vault, Storage Account, MSI) from volatile ones like clusters.

+1 on setting & persisting this during the initial EnvironmentRequest -> DTO -> AzureParameters conversion.

Reply:
To support multi-RG, KeyVaultResourceId will be added as part of CB-11927.
As a workaround, using single resource group and keyVault should be present in the same resource group as mentioned in environment creation request.